docs/en/status-communities/about-losing-access-to-a-community.md

```diff
@@ -0,0 +1,37 @@
+---
+id: 301
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# About losing access to a community
+
+You can lose your Status Community membership for different reasons. For example, a [token-gated community][about-status-communities] can change the token requirements to join, or your behaviour breaks the community rules.
+
+The community owner can permanently ban your account if you break the community rules. Respect the community rules and members, don't spam people and follow the law.
+
+## The basics
+
+- If you lose access to a community, you must join or request to [join the community][join-a-status-community] again.
+- For token-gated communities, you should hold the token requirements at all times.
+- When you lose access to a community, it does show up under the Joined tab on your [Communities page][how-to-use-communities-your-quick-start-guide].
+- In the [notifications centre][view-notifications-and-updates], your Membership notifications show when you have lost access to a community. You don't receive a notification when you're banned from a community.
+
+!!! info
+
+    Community owners control who has access to their communities. Status can't kick you out or ban you from a community. Likewise, status can't grant you access to a community or override the community owner's actions.
+
+## Reasons you can lose access to a community
+
+There are four reasons:
+
+| Reason | Description | Scope |
+|:---|:---|:---|
+| Kick | The community owner has kicked you out of the community. | Open and token-gated communities |
+| Ban | The community owner has permanently banned you from the community. You can't request to join the community again. | Open and token-gated communities |
+| Token requirements change | The token requirements to access the community have changed and your tokens do not longer satisfy these requirements. Open the community page to review the new token requirements. | Token-gated communities |
+| Your tokens change | Your tokens or token amounts have changed and you don't longer satisfy the community requirements. Open the community page to verify your eligibility. | Token-gated communities |
```

Comment:
Not sure if worth adding here as a point as this could be considered an edge case, but so that this has been raised somewhere:

Another way to loose access to a community is if a community owner hasn't backed up the private key of the community and then re-imports her account at some point. The owner is then locked out of their own community.